setup.py

```diff
@@ -50,6 +50,6 @@
     },
     # TODO in the near future
     # python_requires=">=3.6, <4",
-    python_requires=">=3.6, <4",
+    python_requires=">=3.7, <4",
```

Comment:
If you have time you might start adding a pyproject.toml file 
as it is recommended now (you can check with Qeuntin or me  or look at other repos)

I think pypeman, mh_lib and others sue alerady pyproject.toml

Reply:
no time ;)